### PR TITLE
Added an option to copy the connection string

### PR DESCRIPTION
**Pull request by UtechtDustin**
As we have a option to import a connection string we also should have a option to copy that connection string (also usefull for e.g. pdo (PHP) ).

The `hr` will cause a horizontal scrollbar, im not sure whats the best way to add a seperator with `x-menuitem`.

### CODE REVIEW

File: src/components/sidebar/connection/ConnectionListItem.vue [hunk 1]
```diff
@@ -101,6 +110,14 @@ export default {
     },
     duplicate() {
       this.$emit('duplicate', this.config)
+    },
+    copyUrl() {
```
Comment (rathboma) [inline src/components/sidebar/connection/ConnectionListItem.vue:114]: If you make this an `async` function you can use `await this.$copyText` then just try/catch as usual

File: src/components/sidebar/connection/ConnectionListItem.vue [hunk 2]
```diff
@@ -101,6 +110,14 @@ export default {
...
+      this.close()
+      this.$copyText(this.config.fullConnectionString)
```
Comment (rathboma) [inline src/components/sidebar/connection/ConnectionListItem.vue:116]: Should this include the password if it is saved?

**Comment by rathboma** [comment]
HR is the best way to do it. We can adjust styles later

File: src/components/sidebar/connection/ConnectionListItem.vue (hunk 2 above)
Comment (UtechtDustin) [reply src/components/sidebar/connection/ConnectionListItem.vue:116]: No, the same getter will be used for the `title` attributes of the connection list item.

File: src/components/sidebar/connection/ConnectionListItem.vue (hunk 1 above)
Comment (UtechtDustin) [reply src/components/sidebar/connection/ConnectionListItem.vue]: Good to know, thanks!
I changed that.

**Comment by UtechtDustin** [comment]
> HR is the best way to do it. We can adjust styles later

Okay, i was not sure if there is maybe already a element/component for that.

**Comment by rathboma** [comment]
So good.

Vue tip: in templates you can just use 'connectionType' without the 'this'.

**Pull request merged**